### ISSUE TITLE
GPU memory purging

### ISSUE THREAD

**Issue by dominion2** [lllyasviel/stable-diffusion-webui-forge#1937]
It seems that when I load all the parameters for creating a SDXL character the GPU memory builds up after a while.  From the normal stable diffusion A1111, the GPU memory would be purged after a image is generated, however it seems that GPU memory keeps on building up until there are errors.  I need to restart my machine to clear it.  I attempted a manual clearing of the portion of memory that is built up without success.

**Comment by byteconcepts** [comment]
Indeed, that GPU memory isn't freed completely after a "Generate" run is a huge problem.
Even with 24G of GPU memory the GPU is practically completely blocked after a "Generate" run, so it's for example not possible to use even a smaller LLM in local ollama server to let it optimize prompts or answer other kinds of questions.

Even if the actual plan is to keep as much cached as possible, forge ui needs to get a configuration option that let us force a GPU memory freeing after each "Generate" run.

Despite all the other problems that the actual GPU memory management has like not freeing memory when it should, segmentation faults from time to time, even corrupted pycache files after "Generate" runs, even if the run before was successful.
